Week-23/todo-list/Controller/TodoController.js

```diff
@@ -0,0 +1,71 @@
+const Todo = require("../Model/TodoModel");
+exports.postTodo = async (req, res) => {
+  try {
+    const { todo } = req.body;
+    const newTodo = new Todo({ todo });
+    await newTodo.save();
```

Comment:
```suggestion
    const newTodo = await Todo.create(todo);
```
we can directly use create() as it uses save() behind the scenes. Only use save() instead of create() when you need further updates before finally saving in db. Check the mongoose docs for clearly understanding the difference.